umbraco-cloud/set-up/project-settings/README.md

```diff
@@ -11,13 +11,15 @@ When working with a Umbraco Cloud project, you can handle a lot of the project c
 * [Public Access](./#public-access)
 * [Deployment Webhooks](./#webhooks)
 * [Secure Sockets Layer (SSL) certificates](./#certificates-only-available-on-professional-or-enterprise-plan)
-* [CI/CD Flow.](./#umbracocicd)
```

Comment:
@jesp209i didn't notice this at first, but how come you are removing CI/CD from the overview? 🤔